cmd/root.go

```diff
@@ -56,6 +56,7 @@ func Execute(ctx context.Context, version string) {
 	rootCmd.Flags().StringVarP(&input.eventPath, "eventpath", "e", "", "path to event JSON file")
 	rootCmd.Flags().StringVar(&input.defaultBranch, "defaultbranch", "", "the name of the main branch")
 	rootCmd.Flags().BoolVar(&input.privileged, "privileged", false, "use privileged mode")
+	rootCmd.Flags().BoolVar(&input.nodood, "no-dood", false, "do not use DooD")
```

Comment:
do you prefer to make separate defaultEntryPoint, networkingMode and noMapDockerSocket?

Reply:
Hello, thank you for your comments. In my view, I see that `act` is somehow "requiring" DooD (by forcing the bind-mount, the host networking and the entrypoint. Un-hard-coding this stuff (and allowing the defaults) would allow to run things inside any containers that start services (even dind). So I guess the sematics would not mean to "enable-dind"